test/DynamoCoreTests/CoreDynTests.cs

```diff
@@ -69,21 +69,38 @@ public void AddSubtractMapReduceFilterBasic()
         [Test]
         public void verifyNodeStates()
         {
-            // open/run xml test graph
+            // Open/Run XML test graph
             string openPath = Path.Combine(TestDirectory, @"core\NodeStates.dyn");
             RunModel(openPath);
 
-            // check dead node
+            // Check dead node XML
             var deadNode = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("1237a148-7a90-489d-b677-11038072c288");
             Assert.AreEqual(ElementState.Dead, deadNode.State);
-
-            // check warning node
+            // Check warning node XML
             var warningNode = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("50219c24-e583-4b85-887c-409fb062da6e");
             Assert.AreEqual(ElementState.Warning, warningNode.State);
-
-            // check active node
+            // Check active node XML
             var activeNode = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("72136fa9-7aec-4ed5-a23b-1ee1c13294a6");
             Assert.AreEqual(ElementState.Active, activeNode.State);
+
+            // Save/Open/Run JSON graph
+            string tempPath = Path.Combine(Path.GetTempPath(), "NodeStates.dyn");
+            CurrentDynamoModel.CurrentWorkspace.Save(tempPath);
```

Comment:
recall that this won't save a view - does that effect this test?

Reply:
I just double checked, you are correct but I don't think it is required for this test